src/include/sof/sof.h

```diff
@@ -24,7 +24,12 @@ struct sa;
 	({const typeof(((type *)0)->member)*__memberptr = (ptr); \
 	(type *)((char *)__memberptr - offsetof(type, member)); })
 
-#define ALIGN(size, align) (((size) + (align) - 1) & ~((align) - 1))
+/* Align the number to the nearest alignment value */
```

Comment:
Is sof-general header the best place for thta macro? Maybe something with memory would be better?

Reply:
It depends how to look at it. It doesn't necessary need to be used for memory allocation.

Reply:
I know it can be used for other things, just wanted to not have maths/logic in sof.h, cos it will be, a trash file soon with all things that don't fit any other place. We have there some generic stuff, some containers utils, some misc things. I don't know what's the best place for that also. I thought  you'll come up with something 😉  Maybe later when we'll have more things there we will move some of them to other file. Not a big deal.